Add new ttl info around aws external ID (#15715)
DataDog/documentation: content/en/integrations/guide/aws-manual-setup.md

1. In the [AWS integration configuration page][1], click **Add AWS Account**, and then select **Manually**.
2. Select `Role Delegation` for the access type and copy the `AWS External ID`. For more information about the external ID, read the [IAM User Guide][2].  

>   **Note:** The External ID will be available and not regenerate for 48 hours unless explicitly changed by a user or another AWS account is added to Datadog during this period. You can come back to the "Add New AWS Account" page within that time period after the Datadog IAM role is created in AWS to complete the process of adding this account without the External ID changing.

Reviewer: ```suggestion
  **Note:** The External ID remains available and is not regenerated for 48 hours, unless explicitly changed by a user or another AWS account is added to Datadog during this period. You can return to the **Add New AWS Account** page within that time period to complete the process of adding an account without the External ID changing.
```
